src/earn/EarnPoolInfoScreen.test.tsx

```diff
@@ -646,42 +658,240 @@ describe('EarnPoolInfoScreen', () => {
         />
       </Provider>
     )
-    fireEvent.press(getByText('earnFlow.poolInfoScreen.withdraw'))
+    fireEvent.press(getByTestId('WithdrawButton'))
     expect(AppAnalytics.track).toHaveBeenCalledWith(EarnEvents.earn_pool_info_tap_withdraw, {
       providerId: 'aave',
       poolId: 'arbitrum-sepolia:0x460b97bd498e1157530aeb3086301d5225b91216',
       poolAmount: '100',
       networkId: 'arbitrum-sepolia',
       depositTokenId: mockEarnPositions[0].dataProps.depositTokenId,
     })
+    // TODO (ACT-1389): Check that this navigates to confirmation screen for Claim & Withdraw
     expect(navigate).toHaveBeenCalledWith(Screens.EarnCollectScreen, {
       pool: { ...mockEarnPositions[0], balance: '100' },
     })
   })
-
-  it('navigates to EarnEnterAmount when partial withdrawals are enabled', () => {
+  it('open WithdrawBottomSheet when Withdraw button pressed, check that expected options exist', () => {
     jest
       .mocked(getFeatureGate)
       .mockImplementation(
-        (featureGateName) => featureGateName === StatsigFeatureGates.ALLOW_EARN_PARTIAL_WITHDRAWAL
+        (gate) =>
+          gate === StatsigFeatureGates.ALLOW_EARN_PARTIAL_WITHDRAWAL ||
+          gate === StatsigFeatureGates.SHOW_POSITIONS
       )
-
-    const { getByText } = render(
+    const { getByTestId } = render(
       <Provider store={getStore()}>
         <MockedNavigator
           component={EarnPoolInfoScreen}
           params={{
-            pool: { ...mockEarnPositions[0], balance: '100' },
+            pool: {
+              ...mockEarnPositions[0],
+              balance: '100',
+            },
           }}
         />
       </Provider>
     )
-    fireEvent.press(getByText('earnFlow.poolInfoScreen.withdraw'))
+    fireEvent.press(getByTestId('WithdrawButton'))
+    expect(getByTestId('Earn/WithdrawBottomSheet')).toBeVisible()
+    expect(getByTestId('Earn/ActionCard/Withdraw')).toBeTruthy()
+    expect(getByTestId('Earn/ActionCard/Claim')).toBeTruthy()
+    expect(getByTestId('Earn/ActionCard/Exit')).toBeTruthy()
+  })
+  it('tapping withdraw on WithdrawBottomSheet navigates to enter amount screen', () => {
```

Comment:
this file is getting too big, I've started breaking EarnPoolInfoScreen into multiple files on https://github.com/valora-inc/wallet/pull/6176, haven't broken out tests yet. But would it be possible to have all the bottom sheet specific tests in a `WithdrawBottomSheet.test.tsx`? and then `EarnPoolInfoScreen.test.tsx` can have a single test to ensure bottom sheet is opened.

Reply:
@satish-ravi yeah I can do that, for having the bottom sheet tests in their own file, would I still render the pool info screen and then have it tap to open the bottom sheet, or is there a way to render just the bottoms sheet?

Reply:
moved to their own file in [5ceadc2](https://github.com/valora-inc/wallet/pull/6142/commits/5ceadc24807e5decc9ad1669180c0bd1115cd88f), did it similar to `EarnDepositBottomSheet.test.tsx` where I could just render the bottom sheet and seems to work as expected!